packages/mongodb_atlas/data_stream/project/manifest.yml

```
@@ -0,0 +1,63 @@
+title: Collect Project logs from MongoDB Atlas
+type: logs
+streams:
+  - input: cel
+    title: MongoDB Atlas Project logs
+    description: Collect Project logs from MongoDB Atlas.
+    enabled: false
+    template_path: input.yml.hbs
+    vars:
+      - name: groupId
+        type: text
+        title: Group ID
+        description: Identifier string that identifies your project. Groups and projects are synonymous terms. Your group ID is the same as your project ID. Ex. 32b6e34b3d91647abb20e7b8
+        multi: false
+        required: true
+        show_user: true
+      - name: interval
+        type: text
+        title: Interval
+        description: Duration between requests to the Atlas API. Supported units for this parameter are h (hours), m (minutes) and s (seconds).
+        default: 5m
+        multi: false
+        required: true
+        show_user: false
+      - name: http_client_timeout
+        type: text
+        title: HTTP Client Timeout
+        description: Duration before declaring that the HTTP client connection has timed out. Valid time units are h (hours), m (minutes) and s (seconds). If the data collection period is extensive, ensure that this parameter's value is appropriately long.
+        multi: false
+        required: true
+        show_user: false
```

```suggestion
        required: true
        show_user: false
```

It is a required field but not shown to the user. I think `required: false` is the correct one? You already have the default value.

---

Addressed!!